doordeck-sdk/src/androidMain/kotlin/com/doordeck/multiplatform/sdk/model/responses/LockResponses.android.kt

```diff
@@ -176,77 +202,86 @@ internal fun BasicLocationRequirementResponse.toLocationRequirementResponse(): L
     accuracy = accuracy
 )
 
+@JvmSynthetic
 internal fun BasicUnlockBetweenSettingResponse.toUnlockBetweenSettingResponse(): UnlockBetweenSettingResponse = UnlockBetweenSettingResponse(
-    start = start,
-    end = end,
-    timezone = timezone,
-    days = days,
-    exceptions = exceptions
+    start = start.toLocalTime(),
+    end = end.toLocalTime(),
+    timezone = timezone.toZoneId(),
+    days = days.map { DayOfWeek.valueOf(it.name) }.toEnumSet(),
+    exceptions = exceptions.map { it.toLocalDate() }
 )
 
+@JvmSynthetic
 internal fun BasicLockStateResponse.toLockStateResponse(): LockStateResponse = LockStateResponse(
     connected = connected
```

Comment:
I can't recall if I've mentioned, but `connected` can be null indicating the status is unknown

Reply:
Done, thanks!